src/main/java/nextstep/courses/domain/vo/session/Students.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain.vo.session;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Students {
+    private final List<NsUser> students;
+
+    public Students(NsUser... students) {
+        this(List.of(students));
+    }
+
+    public Students(List<NsUser> students) {
+        this.students = new ArrayList<>(students);
+    }
+
+    public void add(NsUser student) {
+        this.students.add(student);
+    }
+
+    public int size() {
+        return Collections.unmodifiableList(students).size();
```

Comment:
```suggestion
        return students.size();
```

unmodifiableList 는 읽기 전용 콜렉션으로 반환하는 동작을 합니다.
size를 반환할 때 굳이 새로운 List를 만들 필요는 없어보이네요.

Reply:
확인했습니다. 이미 읽기전용이었는데 불필요한 래핑이 한번 더 들어갔네요😅

수정해서 다음 PR에 올리겠습니다😀